crates/fluvio-cluster/src/start/local.rs

```diff
@@ -658,18 +602,20 @@ impl LocalInstaller {
 
         config_file.save()?;
 
-        self.render(InstallProgressMessage::ProfileSet);
+        self.render(InstallProgressMessage::ProfileSet, None);
 
         Ok(format!("local context is set to: {}", local_addr))
     }
 
     #[instrument(skip(self))]
     async fn launch_spu_group(&self, client: SharedK8Client) -> Result<(), LocalInstallError> {
         let count = self.config.spu_replicas;
-        self.render(InstallProgressMessage::LaunchingSPUGroup(count));
+
```

Comment:
Instead of having everyone managing progress bar.  just a create a global progress bar at beginning since everything is a single sequential.    then anytime, we want to make show status, it should be just matter of:
```
self.progress.println("creating management group");    // this will print new line 'creating management group' and show spinner.
self.progress.println("set up profile");   // this will print new line and show spinner
self.progress.println_timer("Setting up SPU {I}",i);   // this will print spinner but display count time
```